draft-ietf-quic-recovery.md

```diff
@@ -1318,8 +1318,8 @@ window.
 
 ~~~
    CongestionEvent(sent_time):
-     // Start a new congestion event if the sent time is larger
-     // than the start time of the previous recovery epoch.
+     // Start a new congestion event if packet was sent after the
+     // start of the previous congestion event.
```

Comment:
I understand what the start and the end of an epoch are, but what is the start and end of a congestion event.
In my understanding, a congestion event is when a packet is acknowledged or when a packet is declared lost, and there’s no start and end to it.

Reply:
Good point.  We don't use epoch anywhere else in this doc except this pseudocode, and it's different from the definition in security contexts, so I'd like to avoid the term.

Do you have any other suggestions?  Congestion Period, etc?  I'm not even that happy about using InRecovery here, since in TCP that has a distinct meaning that doesn't map 1:1 to QUIC.

Reply:
Maybe keep congestion event as the trigger. Isn't the period referred to as the recovery period, if entered.

Reply:
Well, seems like you just made a commit doing exactly that ...

Reply:
PTAL at https://github.com/quicwg/base-drafts/pull/2588

Reply:
I did, we keep postposting.